atlp-rwanda/rca-barefoot-team6-bn
# Ft login with google and facebook

#### What does this PR do?
 
Added login with facebook and google functionalities

#### Description of the task to be completed?



#### How should this be manually tested?

Check out this branch, run `npm install`, 
Make sure your .env have all properties in `.env.example`
then go to `{BASE_URL}/api/users/auth/google` or `{BASE_URL}/api/users/auth/facebook` and then you can login in with your credentials. 

#### Any background context you want to provide?

None

#### What are the relevant pivotal tracker stories?

# 184638374

#### Screenshots (if appropriate)

...
